styles.css

```diff
@@ -806,13 +703,338 @@ button:active {
   background: #e05304;
 }
 
+@media (max-width: 768px) {
+  .hamburger {
+    cursor: pointer;
+  }
+
+  .hamburger.active {
+    width: 40px;
+    height: 30px;
+  }
+
+  .bar {
+    display: block;
+    width: 25px;
+    height: 3px;
+    margin: 5px;
+    margin-right: 15px;
+    -webkit-transition: all 0.3s ease;
+    transition: all 0.3s ease;
+    background-color: white;
+  }
+
+  .hamburger.active .bar:nth-child(2) {
+    opacity: 0;
+  }
+
+  .hamburger.active .bar:nth-child(1) {
+    transform: translateY(8px) rotate(45deg);
+    background-color: #67798e;
+  }
+
+  .hamburger.active .bar:nth-child(3) {
+    transform: translateY(-8px) rotate(-45deg);
+    background-color: #67798e;
+  }
+
+  span.bars.active {
+    width: 10px;
+    height: 10px;
+  }
+
+  .ul-nav {
+    position: fixed;
+    left: -100%;
+    display: flex;
+    top: 10px;
+    padding-top: 20px;
+    gap: 24px;
+    flex-direction: column;
+    background: #3c3a39;
+    width: 89.4%;
+    height: 100vh;
+    font-size: 27px;
+    transition: 0.3s;
+    list-style: none;
+    font-family: 'Inter', sans-serif;
+    font-weight: 700;
+  }
+
+  .nav-li {
+    border-bottom: solid 1px #6f6c6b;
+    width: 90%;
+  }
+
+  .nav-link {
+    color: white;
+    margin-left: 20px;
+    width: 80%;
+  }
+
+  .ul-nav.active {
+    left: 0;
+  }
+
+  .nav-mail {
+    display: none;
+  }
+
+  .container {
+    position: fixed;
+    height: 55px;
+    display: flex;
+    place-content: space-between;
+    align-items: center;
+    width: 100%;
+    background-color: #3c3a39;
+  }
+
+  .name {
+    text-decoration: none;
+    color: var(--white-color);
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-size: 18px;
+    line-height: 24px;
+    height: 24px;
+    padding-left: 15px;
```

Comment:
- [ ] You really did a great job on the design but you need to fix the below issue and make sure it matches with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design.
  - Please reduce the gap between project title and technologies.

### Your design
![image](https://user-images.githubusercontent.com/83803414/195177162-1e3e5e30-1e9b-4d4f-a3b1-81cd48ac8db3.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/195177272-9bef7d22-0138-4d55-ba9f-81daaf2bee3a.png)